plenum/test/client/test_client_authn.py

```diff
@@ -58,7 +59,7 @@ def testMessageModified(sa, cli, msg, sig):
     # slight modification to the message
     msg2['myMsg'] = msg2['myMsg'][:-1] + '!'
 
-    with pytest.raises(InvalidSignature):
+    with pytest.raises(InsufficientCorrectSignatures):
```

Comment:
Why do we expect `InsufficientCorrectSignatures` here? If we have just one signature, I think `InvalidSignature` looks more natural.